components-api/src/test/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidatorSpec.scala

```diff
@@ -13,10 +15,12 @@ class ParameterValidatorSpec extends AnyFunSuite with TableDrivenPropertyChecks
     forAll(
       Table(
         ("inputExpression", "isValid"),
-        ("''", false),
-        (" '' ", false),
```

Comment:
Why did you remove these examples with `''`?

Reply:
Because now we pass the evaluated object to validators, not SpEL expression. Be aware that `"''"` as the expression is a blank string, but as the evaluated object - it is not. So these test cases now return `true`.